README.rst

```diff
@@ -0,0 +1,162 @@
+   automatic sanity check using a few example programs. For this to work
+   correctly, you will need python, and ``python-devel`` packages installed on your
+   system. See ``doc/prereqs.rst`` for more information.
+
+        ``gmake check``
+
+   or:
+
+        ``make check``
+
+
+6) Compile an example program using:
+
+        ``chpl -o hello examples/hello.chpl``
+
+
+7) Execute the resulting executable:
+
+       ``./hello``
+
+
+8) Experiment with Chapel in Quick Start mode to your heart's content.
+
+   Once you are comfortable with Chapel and interested in using a
+   full-featured version in the preferred configuration:
+
+   a) Open up a new shell to avoid inheriting the previous environment
+      settings.
+
+   b) Repeat steps 1-7 above, but in Step 2, use ``util/setchplenv.*``
+      instead of ``util/quickstart/setchplenv.*``
+
+   This will set up your environment to use Chapel in the preferred
+   configuration.  Making this configuration will involve building one
+   or more third-party packages which will increase the overall build
+   time.  If you run into any portability issues, please let us know
+   at chapel_info@cray.com.
+
+
+9) If you plan to do performance studies of Chapel programs, be sure
+   to (a) use the full-featured version and (b) read the ``PERFORMANCE``
+   file in this directory to avoid common pitfalls.
+
+What's next?
+------------
+=============================================== =====================================
+**For more detailed information about:**        **refer to:**
+----------------------------------------------- -------------------------------------
+    changes since the last release              ``CHANGES``
+    implementation status                       ``STATUS``
+    performance status                          ``PERFORMANCE``
+    Chapel's file and directory structure       ``README.files``
+    prerequisites for using this release        ``doc/prereqs.rst``
+    setting Chapel environment variables        ``doc/chplenv.rst``
+    building the compiler                       ``doc/building.rst``
+    using the compiler                          ``doc/compiling.rst``
+    executing Chapel programs                   ``doc/executing.rst``
+    debugging Chapel programs                   ``doc/debugging.rst``                  
+    reporting bugs                              ``doc/bugs.rst``
+    Chapel modes for emacs and vim              ``etc/README.md``
+    example Chapel programs                     ``examples/README``
+    a quick reference for Chapel syntax         ``doc/quickReference.pdf``
+    the Chapel language specification           ``doc/chapelLanguageSpec.pdf``
+    executing programs on multiple locales      ``doc/multilocale.rst``
+    how Chapel tasks are mapped to threads      ``doc/tasks.rst``
+    calling C code from Chapel                  ``doc/technotes/extern.rst``
+    other user and technical notes              ``doc/README``
```

Comment:
This too seems like it could go away with the online docs.  Or else, maybe we should point to the root of the online Chapel documentation somewhere in this table?